specificatie/referentie-data-api/resolved/openapi.yaml

```diff
@@ -206,6 +206,9 @@ components:
           type: string
         minderjarige:
           $ref: '#/components/schemas/Minderjarige'
+        inOnderzoek:
+          type: boolean
```

Comment:
doen we hier in de dataservice al een boolean in plaats van de waarde uit de database?

Reply:
de data service kopt door wat hij van de gezag api krijgt of gaat de gezag api zelf een eigen magic number fabriceren uit de mogelijke aanduidingInOnderzoek waarden?

Reply:
je hebt gelijk, ik had niet door dat dit zo uit de gezag api komt